src/diffusers/loaders.py

```diff
@@ -943,14 +943,19 @@ def _modify_text_encoder(self, attn_processors: Dict[str, LoRAAttnProcessor]):
                 module = self.text_encoder.get_submodule(name)
                 # Construct a new function that performs the LoRA merging. We will monkey patch
                 # this forward pass.
-                lora_layer = getattr(attn_processors[name], self._get_lora_layer_attribute(name))
+                attn_processor_name = ".".join(name.split(".")[:-1])
```

Comment:
To get the correct mapping in the names as discovered in https://github.com/huggingface/diffusers/pull/3437#issuecomment-1553281046